docs/reference/tools-reference/new-llm-tool.md

```diff
@@ -0,0 +1,84 @@
+| echo                   | boolean     | value that indicates whether to echo back the prompt in the response. Default is false. | No       |
+| presence\_penalty      | float       | value that controls the model's behavior with regards to repeating phrases. Default is 0.                              | No       |
+| frequency\_penalty     | float       | value that controls the model's behavior with regards to generating rare phrases. Default is 0.                             | No       |
+| best\_of               | integer     | the number of best completions to generate. Default is 1.                               | No       |
+| logit\_bias            | dictionary  | the logit bias for the language model. Default is empty dictionary.                     | No       |
+
+
+### Chat
+
+
+| Name                   | Type        | Description                                                                                    | Required |
+|------------------------|-------------|------------------------------------------------------------------------------------------------|----------|
+| prompt                 | string      | text prompt that the language model will response                                              | Yes      |
+| model, deployment_name | string      | the language model to use                                                                      | Yes      |
+| max\_tokens            | integer     | the maximum number of tokens to generate in the response. Default is inf.                      | No       |
+| temperature            | float       | the randomness of the generated text. Default is 1.                                            | No       |
+| stop                   | list        | the stopping sequence for the generated text. Default is null.                                 | No       |
+| top_p                  | float       | the probability of using the top choice from the generated tokens. Default is 1.               | No       |
+| presence\_penalty      | float       | value that controls the model's behavior with regards to repeating phrases. Default is 0.      | No       |
+| frequency\_penalty     | float       | value that controls the model's behavior with regards to generating rare phrases. Default is 0.| No       |
+| logit\_bias            | dictionary  | the logit bias for the language model. Default is empty dictionary.                            | No       |
+| tool\_choice           | object      | value that controls which function is called by the model. Default is null.                    | No       |
+| tools                  | list        | a list of tools the model may generate JSON inputs for. Default is null.                       | No       |
+| response_format        | object      | an object specifying the format that the model must output. Default is null.                   | No       |
+
+## Outputs
+
+| API        | Return Type | Description                              |
```

Comment:
No need to distinct by api. 


| Return Type | Description                              |
|-------------|------------------------------------------|
| string      | The text of one predicted completion or response of conversation |

Reply:
fixed